_drafts/contribute-to-ansible.markdown

```diff
@@ -16,9 +16,32 @@
     git installé
 
 ## What's Ansible?
-    Principe de fonctionnement
-    petit exemple d'installation d'un paquet (liens vers le site officiel de ansible)
-    Différencier serveur de client (machine hote ou cible) retrouve rles vrais termes ansible
+    
+If you are reading this article, you should propably be already familiar with Ansible... So if you're not reading it, we'll explain it quickly ...
+    
+Why Ansible ? Because it's cool! But not only ... It's really really more practical and easy to use than just cool. Ansible will help you to manage easily your laptops/servers/network devices/IoT/everything or almost ! Ansible force lie in the capability to easily configure a lot of devices and guaranty that the configuration is consistent in all targets regardless the initial state. How is it done ? The concept behind an Ansible module is to be re-enterable!
```

Comment:
Why Ansible?
But not only...
almost!
done?

Reply:
Talk about provisionning tools and make a short compare with other solutions like puppet/chef

Reply:
provisionning tools ? What do you think about ?

Reply:
make an quickly introduce to the provisionning tools concept.
Ansible as an provisionnig tools. Provisionnig tools allow to you to manage simply your machine maintenance. With these tools you can drive machine management in an industrial way.

Others provisionning tools exist, like puppet / chef but their are out of the scope of this post.